public/data/benjaminmichaelis.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "Benjamin Michaelis",
+  "type": "personal",
+  "bio": "Full time student, part-time adventure seeker/traveler",
```

It seems more attractive

```suggestion
  "bio": "Full time Student | Part-time Adventure seeker | Traveler",
```